sdk/src/main/scala/io/horizen/AbstractSidechainNodeViewHolder.scala

```diff
@@ -282,7 +282,8 @@ abstract class AbstractSidechainNodeViewHolder[
     if (!history().contains(pmod.id)) {
       context.system.eventStream.publish(StartingPersistentModifierApplication(pmod))
 
-      log.info(s"Apply modifier ${pmod.encodedId} of type ${pmod.modifierTypeId} to nodeViewHolder")
+      log.info(s"Apply modifier ${pmod.encodedId} of type ${pmod.modifierTypeId} to nodeViewHolder, " +
+        s"state dump enabled: ${sidechainSettings.evmStateDump.enabled}")
```

Comment:
why log the setting value here?

Reply:
It is a way to check that `preimages` were enabled for each block. For the dump to succeed it is necessary that the node has the `preimages` enabled from the genesis block, without skipping any block. In the original Jira task, it was required that the `nodeInfo` endpoint returned first and last block with `preimages` enabled, in order to check that all blocks had `preimages`, but it was too much work for a feature that will be used just once. With this tracing, that it is a pork-around actually, a `grep` is enough to check that the node synch'ed correctly for the dump.

Reply:
let's add the string ", state dump enabled:" only if enabled = true
(so that by default we don't add anything more in the logs compared to now)

Reply:
Done